### PR TITLE
feat(): ui input for triggerExecutionId

### PR DESCRIPTION
**Pull request by Skraye**

### CODE REVIEW

**Comment by Skraye** [comment]
close #2489

File: ui/src/components/executions/Executions.vue [hunk 1]
```diff
@@ -61,6 +61,14 @@
                         />
                     </el-select>
                 </el-form-item>
+                <el-form-item>
+                    <el-input
+                        :placeholder="$t('triggerExecutionId')"
```
Comment (brian-mulier-p) [inline ui/src/components/executions/Executions.vue:66]:
```suggestion
                        :placeholder="$t('trigger execution id')"
```

**Review by tchiotludo — COMMENTED** [review]
Seems to be overkilled to add a new filter for this, no ?

**Comment by Skraye** [comment]
As I explained on slack, without a UI to interact with the filter, we cannot reset the triggerExecutionId when coming from the forEach progress bar, because the restoreUrl always set the filter back

**Pull request merged**